### ISSUE TITLE
IdsDatagrid Header Tooltip truncated text

### ISSUE THREAD

**Issue by kenneth-acejo** [infor-design/enterprise-wc#2578]
**Describe the bug**
Tooltip of datagrid header truncates text if it exceeds border of card and accordion.

**To Reproduce**

1. Go to https://stackblitz.com/edit/ids-enterprise-wc-v1-leqiuq?file=main.js
2. Hover on **abc** column header of the first datagrid (inside accordion and card)
3. Tooltip is truncated (if tooltip does not show, I scroll down a bit so that the tooltip is forced to show at the bottom of target)

**Expected behavior**
Tooltip should display full text and not truncated.

**Version**
ids-enterprise-wc@1.2.0

**Screenshots**
<img width="940" alt="datagridtooltip" src="https://github.com/infor-design/enterprise-wc/assets/17284784/e52ba11f-4ca3-42aa-ab54-99fe2f19b906">

Label is team: m3

**Comment by tmcconechy** [comment]
@kenneth-acejo i dont see a tooltip at all on the top grid?

**Comment by kenneth-acejo** [comment]
Yeah, it's kind of intermittent. Could be another bug. But, what has worked for me is I scroll down so that the column header is at the top or some few pixels away from the top. Then when I hover over the header, the tooltip is shown below it. 


![no-scroll-tooltip](https://github.com/user-attachments/assets/ae70bb42-e71f-4996-9ae2-cb88932113fe)

vs

![scroll-tooltip](https://github.com/user-attachments/assets/67c18149-d80a-4a12-9c4f-4ec2e3f1c946)

**Comment by kenneth-acejo** [comment]
Actually, I just added

`tooltipOptions: {
        headerPlacement: 'bottom'
      }`

Without this, maybe a separate issue, it just does not show most of the time. Also, the truncated text is still there.

**Comment by tmcconechy** [comment]
ok added this to the backlog will take a look

**Comment by kenneth-acejo** [comment]
Yeah it still persists in 1.3.0
https://stackblitz.com/edit/ids-enterprise-wc-v1-leqiuq?file=index.html
